Util/SpeciesGrowthEvaluator.py

```diff
@@ -65,22 +86,32 @@ def spread_value(arr: np.ndarray, target: int, grow_chance: float) -> np.ndarray
             for j, val in enumerate(row):
                 if val == target:
                     if i > 0 and random.random() < grow_chance:
-                        out[i-1][j] = val
-                    if i < len(arr)-1 and random.random() < grow_chance:
-                        out[i+1][j] = val
+                        out[i - 1][j] = val
+                    if i < len(arr) - 1 and random.random() < grow_chance:
+                        out[i + 1][j] = val
                     if j > 0 and random.random() < grow_chance:
-                        out[i][j-1] = val
-                    if j < len(arr[0])-1 and random.random() < grow_chance:
-                        out[i][j+1] = val
+                        out[i][j - 1] = val
+                    if j < len(arr[0]) - 1 and random.random() < grow_chance:
+                        out[i][j + 1] = val
         return out
 
-
+    # Create an evaluator and use our test data on it
     evaluator = SpeciesGrowthEvaluator()
     test_map = np.random.randint(low=0, high=3, size=(50, 50))
     for _ in range(4):
         evaluator.add(test_map)
         test_map = spread_value(test_map, 2, 0.2)
-
     evaluator.display_maps()
     evaluator.evaluate()
 
+    # Show the growth of each species over time
```

Comment:
Wait, why did this get moved out of the evaluator?

Reply:
I think it's a bit messy being in there, I only really wanted to show how you could display the data but I don't want users to think that's the only way to use it

Reply:
I think that it would be fine in there if you just explain that, and you could have some sort of argument to toggle the feature if you want, personally I think it's messier to have evaluation stuff outside of the evaluation section, but it is up to you

Reply:
Well the main is only ran when you run the script directly, won't the typical use case be to import the module into our API and run it from there? 

Idk if its standard I just like to have some example code in a main block bc it'll never get run unless you go to the script directly